### PR TITLE
chore: deno compat

### PR DESCRIPTION
**Pull request by harrysolovay**
Much more to do (hence draft mode).

### CODE REVIEW

File: _deps/mod.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// "@polkadot/api": "^8.9.1",
```
Comment (pepoviola) [inline _deps/mod.ts:1]: Hi @harrysolovay I'm not sure the purpose of this file? is something deno related?

File: _deps/polkadot/api.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "https://deno.land/x/polkadot@0.0.4/api/mod.ts";
```
Comment (pepoviola) [inline _deps/polkadot/api.ts:1]: This version of `polkadot` is maintained by you @harrysolovay ?

File: _deps/mod.ts (hunk 1 above)
Comment (harrysolovay) [reply _deps/mod.ts:1]: It's a temporary file that lists out all deps, both dev & runtime. It's useful for us to gradually delete items as we reference the Deno-compatible versions / replacements.

File: _deps/polkadot/api.ts (hunk 2 above)
Comment (harrysolovay) [reply _deps/polkadot/api.ts:1]: No, it's maintained by Jaco.

**Comment by pepoviola** [comment]
Hey @harrysolovay, looks good!! I have a couple of questions and clarifications since I'm not sure how some stuffs works in `Deno`. But is better to make a quick call.

Thanks!

**Comment by pepoviola** [comment]
Closing here @harrysolovay, ping me (or @kratico) if you need some customizations.

Thanks!!

**Pull request closed without merging**